x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/index.tsx

```diff
@@ -591,15 +592,18 @@ const RuleDetailsPageComponent: React.FC<DetectionEngineComponentProps> = ({
                 border
                 subtitle={subTitle}
                 subtitle2={
-                  <>
-                    <EuiFlexGroup gutterSize="xs" alignItems="center" justifyContent="flexStart">
+                  <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexStart">
+                    {rule?.rule_source?.type === 'external' && rule.rule_source.is_customized && (
+                      <EuiBadge color="hollow">{i18n.CUSTOMIZED_PREBUILT_RULE_LABEL}</EuiBadge>
```

Comment:
Let's also extract this into a reusable component and put it in `x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details`.

Reply:
Done!